ambry-store/src/main/java/com.github.ambry.store/Log.java

```diff
@@ -470,17 +466,15 @@ private void ensureCapacity(long writeSize) throws StoreException {
           new LogSegment(segmentNameAndFilename.getFirst(), newSegmentFile, segmentCapacity, metrics, true);
       segmentsByName.put(segmentNameAndFilename.getFirst(), newSegment);
     } catch (StoreException e) {
-      logger.error("Failed to create new log segment {} with store exception: ", segmentNameAndFilename.getFirst(), e);
       try {
```

Comment:
a test for this block will be good, but its ok if its too cumbersome.

Reply:
test added